Projektor.sln

```diff
@@ -19,6 +19,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EventBus", "Common\EventBus
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Services", "Services", "{B81E4035-8BA6-4F07-A747-EAD0B5C7AF75}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Gateway", "Services\Gateway\Gateway.csproj", "{D0E8F3D9-FFD4-478B-BDCB-2B1D065F434A}"
```

Comment:
Neka konvencija je da se gateway nalazi u zasebnom folderu tipa `ApiGateways` umesto u `Services`. Takodje, kako moze da postoji vise gateway-a (za WabApp, Mobile, ...) ovaj projekat bi trebalo preimenovati u nesto kao `WebGateway` ili `WebAppGateway`

Reply:
iskreno, ne umem :D  
Probao sam i u rideru i u VS-u i totalno sam zeznuo solution konfiguraciju. Aj uradi ti ako umes ahahah

Reply:
Nekako sam uspeo da ga premestim, sada vi proverite da li jos uvek radi XD
Usput sam video da je ukljucen https pa da imamo to na umu ako nesto ne bude radilo

Reply:
radi